ansible_collections/arista/avd/roles/eos_l3ls_evpn/templates/documentation/dc-documentation.j2

```diff
@@ -0,0 +1,95 @@
+{% set dc_doc.spine_total = 0 %}
+{% for node in groups[dc_name] | arista.avd.natural_sort %}
+{%     if hostvars[node].type == "spine" %}
+{%     set dc_doc.spine_total = dc_doc.spine_total + 1 %}
+{%     endif %}
+{% endfor %}
+# {{ dc_name }}
+
+## Table of Contents
+
+- [{{ dc_name }}](#{{ dc_name|replace('_',"")|lower }} )
+  - [Spine and Super-Spine Switches and Management IP](#spine-and-super-spine-switches-and-management-ip)
+  - [Super Spine Fabric Topology](#super-spine-fabric-topology)
+  - [Super Spine Fabric IP Allocation](#super-spine-fabric-ip-allocation)
+  - [Super Spine Fabric Point-To-Point Links](#super-spine-fabric-point-to-point-links)
+  - [Point-To-Point Links Node Allocation](#point-to-point-links-node-allocation)
+  - [Super Spine Overlay Loopback Interfaces (BGP EVPN Peering, if required)](#super-spine-overlay-loopback-interfaces-bgp-evpn-peering-if-required)
+  - [Loopback0 Interfaces To Super Spine Node Allocation](#loopback0-interfaces-to-super-spine-node-allocation)
+
+## Spine and Super-Spine Switches and Management IP
+
+| Node | Management IP | Platform |
+| ---- | ------------- | -------- |
+{% for node in groups[dc_name] | arista.avd.natural_sort %}
+{%     if hostvars[node].type == "super-spine" %}
+| {{ node }} | {{ hostvars[node].switch_mgmt_ip }} | {{ hostvars[node].switch_platform }} |
+{%     endif %}
+{% endfor %}
+{% for node in groups[dc_name] | arista.avd.natural_sort %}
+{%    if hostvars[node].type == "spine" %}
+| {{ node }} | {{ hostvars[node].switch_mgmt_ip }} | {{ hostvars[node].switch_platform }} |
+{%     endif %}
+{% endfor %}
+
+## Super Spine Fabric Topology
+
+| Type | Node | Interface | Peer Node | Peer Interface |
+| ---- | --------- | -------------- | --------- | -------------- |
+{% for node in groups[dc_name] | arista.avd.natural_sort %}
+{%     if hostvars[node].type == "super-spine" %}
+{%         if hostvars[node].ethernet_interfaces is defined and hostvars[node].ethernet_interfaces is not none %}
+{%             for ethernet_interface in hostvars[node].ethernet_interfaces | arista.avd.natural_sort %}
+{%                  if hostvars[node].ethernet_interfaces[ethernet_interface].peer_type == 'spine' %}
+| super-spine | {{ node }} | {{ ethernet_interface }} | {{ hostvars[node].ethernet_interfaces[ethernet_interface].peer }} | {{ hostvars[node].ethernet_interfaces[ethernet_interface].peer_interface }} |
+{%                  endif %}
+{%             endfor %}
+{%         endif %}
+{%     endif %}
+{% endfor %}
+
+## Super Spine Fabric IP Allocation
+
+### Super Spine Fabric Point-To-Point Links
+
+| P2P Summary | Available Addresses | Assigned addresses | Assigned Address % |
+| ----------- | ------------------- | ------------------ | ------------------ |
+| {{ super_spine_underlay_p2p_network_summary }} | {{ super_spine_underlay_p2p_network_summary | ipaddr('size') }} | {{ 
+    super_spine.nodes | length * dc_doc.spine_total * 2 * max_spine_to_super_spine_links }} | {{
```

Comment:
super_spine_underlay_p2p_network_summary is not available when this is run for a super-spine. We need to enhance to documentation, so it parse all fabrics like the config templates.